### ISSUE TITLE
Required boilerplate for dev env is complicated

### ISSUE THREAD

**Issue by nonken** [uxebu/bonsai#99]
Not sure if this has been already discussed but the boilerplate for setting up a bonsai dev environment is complicated. without understanding the project there is no way of finding out that you have to include the bootstrapper/_dev/[runner].js - even with docs this looks like a scary script include.

I had expected to being able to include src/bonsai.js ideally without even configuring require (would this even be possible?)

**Comment by davidaurelio** [comment]
Good point. Not sure if this would be possible without even configuring requirejs
